src/map/attackround.cpp

```diff
@@ -199,12 +199,23 @@ void CAttackRound::DeleteAttackSwing()
 void CAttackRound::CreateAttacks(CItemWeapon* PWeapon, PHYSICAL_ATTACK_DIRECTION direction)
 {
     uint8 num = 1;
+    
+    bool isPC = m_attacker->objtype == TYPE_PC;
 
     // Checking the players weapon hit count
     if (PWeapon->getReqLvl() <= m_attacker->GetMLevel())
     {
         num = PWeapon->getHitCount();
     }
+    
+    // If the attacker is a mobentity or derived from mobentity, check to see if it has any special mutli-hit capabilties
+    if (dynamic_cast<CMobEntity*>(m_attacker))
+    {
+        uint8 multiHitMax = (uint8)static_cast<CMobEntity*>(m_attacker)->getMobMod(MOBMOD_MULTI_HIT);
```

Comment:
it's a bit weird to use a c-style cast and a static_cast on the same line... you don't really even need to cast to uint8, just leave it as whatever it is:
auto multiHitMax = static_cast<CMobEntity*>(m_attacker)->getMobMod(MOBMOD_MULTI_HIT);